### DISCUSSION TITLE
V3.beta and register plugin

### DISCUSSION THREAD

**Discussion by Grazulex** [filamentphp/filament#7078] [Help] [answered]
Hello,

I have make my upgrade to the V3.beta version (Filament, Livewire and spatie-laravel-translatable) and I have follow all the step but when I open a table with a "translatable" field I have this error : 

`Plugin [spatie-laravel-translatable] is not registered for panel [admin].

Plugin [spatie-laravel-translatable] is not registered for panel [admin]. (View: /var/www/html/vendor/filament/actions/resources/views/select-action.blade.php) (View: /var/www/html/vendor/filament/actions/resources/views/select-action.blade.php) (View: /var/www/html/vendor/filament/actions/resources/views/select-action.blade.php) (View: /var/www/html/vendor/filament/actions/resources/views/select-action.blade.php) (View: /var/www/html/vendor/filament/actions/resources/views/select-action.blade.php)
/var/www/html/vendor/filament/filament/src/Panel/Concerns/HasPlugins.php#45
`

What can I do to "register" this plugin ?

Thanks you

**Comment by akunbeben** [comment, marked as answer]
See [here](https://beta.filamentphp.com/docs/3.x/panels/plugins), chain the $panel->plugins([])

**Reply by Grazulex** [reply]
Many thanks
